samples/charts/financial-chart/trendlines/Pages/FinancialChartTrendlines.razor

```diff
@@ -0,0 +1,64 @@
+﻿@page "/FinancialChartTrendlines"
+@* this namespace is required for Infragistics controls *@
+@using Infragistics.Blazor.Controls
+@using Samples.Services
+@inject IInfragisticsBlazor InfragisticsBlazor;
+@inject IJSRuntime JSRuntime;
+
+<div class="igContainer">
+    <div class="igOptions">
+        <label class="igOptions-label">Annotations: </label>
+        <label class="igOptions-item">
+            <select @bind="@TrendLineType">
+                <option>@TrendLineType.CubicFit</option>
+                <option>@TrendLineType.LinearFit</option>
+                <option>@TrendLineType.QuinticFit</option>
+                <option>@TrendLineType.QuarticFit</option>
+                <option>@TrendLineType.ExponentialFit</option>
+                <option>@TrendLineType.PowerLawFit</option>
+                <option>@TrendLineType.LogarithmicFit</option>
+                <option>@TrendLineType.CumulativeAverage</option>
+                <option>@TrendLineType.ExponentialAverage</option>
+                <option>@TrendLineType.SimpleAverage</option>
+                <option>@TrendLineType.ModifiedAverage</option>
+                <option>@TrendLineType.WeightedAverage</option>
+                <option>@TrendLineType.None</option>
+            </select>
+        </label>
+        
+    </div>
+
+    <div class="igComponent">
+
+        @if (DataSource != null)
+        {
+            <FinancialChart Width="100%"
+                Height="100%"
+                ChartType=FinancialChartType.Line
+                Thickness=2
+                DataSource="DataSource"
+                TrendLineType="@TrendLineType"
+                TrendLineThickness=2
+                TrendLinePeriod=10
+                TrendLineBrushes="rgba(5, 138, 0, 1), rgba(0, 101, 209, 1)"
+                Brushes="rgba(5, 138, 0, 1), rgba(0, 101, 209, 1)"
+                ChartTitle="Tesla vs Microsoft Trend"
+                Subtitle="Between 2013 and 2017"
+                ZoomSliderType="FinancialChartZoomSliderType.None"/>
+        }
+    </div>
+</div>
+
+@code {
+
+        List<StockItem[]> DataSource;
+        public TrendLineType TrendLineType = TrendLineType.QuinticFit;
+
+        protected override async Task OnInitializedAsync()
+        {
+            base.OnInitialized();
```

Comment:
you cannot call base's OnInitialized from OnInitializedAsync

Reply:
pending